64_FFT/app_resources/shader.comp.hlsl

```diff
@@ -44,8 +61,8 @@ void main(uint32_t3 ID : SV_DispatchThreadID)
 
 	// Workgroup	
 
-	nbl::hlsl::workgroup::FFT<2, true, input_t>::template __call<Accessor, SharedMemoryAccessor>(accessor, sharedmemAccessor);
+	nbl::hlsl::workgroup::fft::FFT<2, true, input_t>::template __call<Accessor, SharedMemoryAccessor>(accessor, sharedmemAccessor);
 	accessor.workgroupExecutionAndMemoryBarrier();
-	//nbl::hlsl::workgroup::FFT<2, false, input_t>::template __call<Accessor, SharedMemoryAccessor>(accessor, sharedmemAccessor);	
+	nbl::hlsl::workgroup::fft::FFT<2, false, input_t>::template __call<Accessor, SharedMemoryAccessor>(accessor, sharedmemAccessor);	
```

Comment:
what's with the double `fft` ?

Reply:
Idk why I added an extra namespace there, I'll try removing it

Reply:
I only wanted it got `exchangeValues` since thats an implementation detail of the FFT (and maybe soon the `bitonic_sort`)